apps/backoffice-v2/src/pages/Entity/hooks/useTasks/useTasks.tsx

```diff
@@ -317,99 +317,103 @@ export const useTasks = ({
+                hideSeparator: true,
+                value: {
+                  title: `${toStartCase(entity?.type)} Information`,
+                  data: [
+                    ...Object.entries(
+                      omitPropsFromObject(entity?.data, 'additionalInfo', 'address') ?? {},
+                    ),
+                    ...Object.entries(entity?.data?.additionalInfo ?? {}),
+                  ]
+                    ?.map(([title, value]) => ({
+                      title,
+                      value,
+                      type: 'string',
+                      isEditable: false,
+                    }))
+                    // removing private properties from list (__kyb_snapshot in this case)
+                    // __kyb_snapshot is state of KYB,temp solution
+                    // payload is not for users so removing it
+                    // TO DO: Remove this as soon as BE updated
+                    .filter(elem => !elem.title.startsWith('__')),
+                },
+              },
             ],
-        };
+          },
+        ];
+
+  const mapBlock =
+    Object.keys(address ?? {})?.length === 0
+      ? []
+      : [
+          {
+            cells: locations &&
+              locations.length && [
+                {
+                  id: 'map-container',
+                  type: 'container',
+                  value: [
+                    {
+                      id: 'header',
+                      type: 'heading',
+                      value: `${toStartCase(entity?.type)} Address`,
+                    },
+                    {
+                      type: 'details',
+                      hideSeparator: true,
+                      value: {
+                        title: `${toStartCase(entity?.type)} Address`,
+                        data:
+                          typeof address === 'string'
+                            ? [
+                                {
+                                  title: 'Address',
+                                  value: address,
+                                  isEditable: false,
+                                },
+                              ]
+                            : Object.entries(address ?? {})?.map(([title, value]) => ({
+                                title,
+                                value,
+                                isEditable: false,
+                              })),
+                      },
+                    },
+                    {
+                      type: 'map',
+                      address,
+                      latitude: locations[0].lat,
+                      longitude: locations[0].lon,
+                    },
+                  ],
+                },
+              ],
+          },
+        ];
 
   return useMemo(() => {
-    return entity ? [entityInfoBlock, ...registryInfoBlock, ...taskBlocks, mapBlock] : [];
+    return entity ? [...entityInfoBlock, ...registryInfoBlock, ...taskBlocks, ...mapBlock] : [];
```

Comment:
Making sure there are no empty objects in this array. This case is spreading an empty array.